libraries/common/tests/auth/test_permissions.py

```
@@ -0,0 +1,383 @@
+        self,
+    ) -> None:
+        """JWT with auth_method=client_credentials loads SA."""
+        # `imbi.common`, not `imbi.api`: this suite must not require imbi-api
+        # to be installed, and reaching into the API's settings singleton
+        # inverts the dependency direction the extraction was for. Matches
+        # what the sibling `test_authorization.py` does.
+        from imbi.common import settings
+        from imbi.common.auth import core
+
+        auth_settings = settings.Auth(
+            jwt_secret='test-secret-key-32-characters!',
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "jwt_secret"

(S106)